runtime/knative/knative.go

```diff
@@ -99,7 +99,7 @@ func (r *Runtime) RegisterHTTPFunction(
 
 		rm.ProcessPreHooks()
 
-		rm.FuncContext.Error = fn(w, r)
+		rm.FuncContext.Error = fn(*rm.FuncContext.SyncRequestMeta.ResponseWriter, rm.FuncContext.SyncRequestMeta.Request)
```

Comment:
Passing the [Context](https://github.com/arugal/functions-framework-go/blob/dev/plugin/skywalking/sync-request.go#L20) of `ExecPreHook` to the user function requires this modification, any good suggestions? @benjaminhuo @tpiperatgod

Reply:
It looks good to me, they're the same.
@tpiperatgod what do you think?

Reply:
lgtm, and we can make some optimizations to the `Context` in the PRs later.

Reply:
@arugal you can change this directly if it is blocking this PR